specification/mediaservices/resource-manager/Microsoft.Media/stable/2020-05-01/streamingservice.json

```diff
@@ -1744,7 +1989,7 @@
     },
     "StreamingEndpointName": {
       "name": "streamingEndpointName",
-      "description": "The name of the StreamingEndpoint.",
+      "description": "The name of the streaming endpoint, maximum length is 24.",
```

Comment:
Funny and strange that the event name is 32, an the endpoint name is only 24 chars in length. I'm sure that this is related to DNS for the Streaming endpoint?

Reply:
Look like its by design.